tools/cloud-function-automation/cloud_function_automation.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/python3
```

Comment:
`#!/usr/bin/env python3`

Python might be in a different location to `/usr/bin` (for instance, if a virtualenv is used).